tstest/integration/integration.go

```diff
@@ -43,38 +43,39 @@ import (
 	"tailscale.com/version"
 )
 
-// Binaries are the paths to a tailscaled and tailscale binary.
-// These can be shared by multiple nodes.
+var (
+	binariesOnce sync.Once
+	binaries     Binaries
+)
+
+// TestBinaries returns tailscale and tailscaled binaries.
+// The returned binaries are valid for the duration of the test.
+func TestBinaries(tb testing.TB) Binaries {
+	binariesOnce.Do(func() {
+		bindir := tb.TempDir()
```

Comment:
But TempDir is removed automatically when a test ends, no?

how does this work with two tests running in serial, both using this?

Reply:
oh right. duh. that's why i did it that way. (that's what I get for letting the pr rot for so long.)

how did you want this to work with sync.Once, then? provide a global cleanup function that clients call from TestMain after running tests?

Reply:
Did the global cleanup function thing. PTAL.